users/views.py

```diff
@@ -13,9 +17,54 @@ class UserRegistrationView(APIView):
     def post(self, request):
         serializer = UserRegistrationSerializer(data=request.data)
         if serializer.is_valid():
-            user = serializer.save()
-            return Response({"message": "User registered successfully"}, status=status.HTTP_201_CREATED)
+            user=serializer.save()
+            user_data= {
+                
+                'email':user.email,
+                
+
+            }
+            encoded_user_data = base64.urlsafe_b64encode(json.dumps(user_data).encode()).decode()

```

Comment:
This token has no lifetime . which means user can click on this any number of times and get verify. So it should be valid only the 24 hrs .